packages/legend-graph/src/graph-manager/__tests__/ImportResolutionRoundtrip.test.ts

```diff
@@ -249,3 +250,14 @@ describe(unitTest('DSL Data import resolution roundtrip'), () => {
     },
   );
 });
+
+describe(unitTest('SubTypeGrpahFetchTree roundtrip'), () => {
+  test.each([
```

Comment:
this file is about testing `import` 
does this roundtrip test have import ?
otherwise, could we add this in the roundtrip-grammar/cases?

Reply:
No this does not have any import, I have added test in roundtrip-grammar/cases